jef/streetmerchant
# feat(discord): dynamic currency symbol

### Description

- Fixes the default currency used for Discord notifications ($) to be store specific
- Added currency as a requirement for each store to define the correct symbol to be shown
- Refactored every store to show the name at the top rather than the bottom

### Testing

1. Set a discord notification (e.g. webhook)
2. Check for an in stock item at any store of your choosing
3. Verify the correct currency symbol is shown in the discord notification

Fixes issue: https://github.com/jef/streetmerchant/issues/1318
